src/select.ts

```diff
@@ -42,13 +42,13 @@ export class FormSelect extends Component<FormSelectOptions> {
   dropdown: Dropdown;
   /** The select wrapper element. */
   wrapper: HTMLDivElement;
-  selectOptions: (HTMLOptionElement|HTMLOptGroupElement)[];
+  selectOptions: (HTMLOptionElement | HTMLOptGroupElement)[];
   private _values: ValueStruct[];
 
   constructor(el: HTMLSelectElement, options: FormSelectOptions) {
     super(el, options, FormSelect);
     if (this.el.classList.contains('browser-default')) return;
-    (this.el as any).M_FormSelect = this;
+    this.el['M_FormSelect'] = this;
```

Comment:
I think we might need to revert this, since it introduces backwards compatibility issues with applications using the previous object orientated approach, propose to just set it to warnings until we can declare the correct type definition